### ISSUE TITLE
Unrelated paths are diffed relatively

### ISSUE THREAD

**Issue by 9999years** [Manishearth/pathdiff#13]
When two completely unrelated paths are diffed, we get back a relative path:

```rust
assert_eq!(
    pathdiff::diff_paths("/a/b/c", "/d/e/f"),
    Some("../../../a/b/c".into())
);
```

This is _longer_ than the input path, although it is relative, but this isn't what I want for displaying a path to a user nicely.

I'd really like an option to disable this behavior — maybe we could introduce a `pathdiff::Options` type?

Otherwise, if this is the intended behavior, we should document it.

**Comment by Manishearth** [comment]
Yes, this is intentional. I don't think "disable this behavior" is as straightforward as it sounds; there's no clear heuristic for when a relative path should be used vs an absolute one: I can think of multiple heuristics and reasons to choose different ones and none of them are universally good. Happy to accept a PR documenting this.

**Comment by 9999years** [comment]
Oh, I actually already added a doctest for this in #11; would you mind cutting a release so it shows up on docs.rs/pathdiff? Thanks so much!

**Comment by Manishearth** [comment]
Published 0.2.2. Thanks!
